src/mscorlib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -582,6 +582,12 @@
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\FileApiInterop\Interop.CreateFile2FromApp.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Kernel32\Interop.CREATEFILE2_EXTENDED_PARAMETERS.cs" />
   </ItemGroup>
+  <ItemGroup Condition="$(TargetsWindows) or '$(FeaturePal)'=='true'">
+    <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Interop.Errors.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Interop.Libraries.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Kernel32\Interop.FormatMessage.cs" />
```

Comment:
I think there will be more ifdefs required to make this work ... we can wait for the CI results.